l10n_it_central_journal/wizard/print_giornale.py

```diff
@@ -51,6 +50,15 @@ def _get_journal(self):
                                    'Target Move', default='all')
     fiscal_page_base = fields.Integer('Last printed page', required=True)
     start_row = fields.Integer('Start row', required=True)
+    year_footer = fields.Char(string='Year for Footer',
```

Comment:
This is a new functionality and has nothing to do with migration, please do this in a separate commit